hiroaki-junit5/gradle-mvn-push.gradle

```diff
@@ -0,0 +1,92 @@
+apply plugin: 'maven'
```

Comment:
I just copy pasted this since it was already. It would be good if you can take a look at this part.

Reply:
Looks good, we need this for deploying. This repo has two different configurations of the maven plugin, one for Android and another one for Kotlin modules. Maybe we could extract both into `/gradle` (root did)  and reference from there for reusability.

Reply:
Will do 👍